src/features/farming/animals/components/Chicken.tsx

```diff
@@ -44,20 +46,18 @@ const getPercentageComplete = (fedAt?: number) => {
 
   const timePassedSinceFed = Date.now() - fedAt;
 
-  if (timePassedSinceFed >= CHICKEN_FEEDING_TIME) return 100;
+  if (timePassedSinceFed >= CHICKEN_TIME_TO_EGG) return 100;
 
-  return Math.ceil((timePassedSinceFed / CHICKEN_FEEDING_TIME) * 100);
+  return Math.ceil((timePassedSinceFed / CHICKEN_TIME_TO_EGG) * 100);
 };
 
 interface TimeToEggProps {
-  timeToEgg: number;
+  service: MachineInterpreter;
   showTimeToEgg: boolean;
 }
 
-const TimeToEgg = ({ timeToEgg, showTimeToEgg }: TimeToEggProps) => {
-  const [timeLeft, setTimeLeft] = useState(timeToEgg);
-  // This interval will only run when this is showing in the ui
-  useInterval(() => setTimeLeft((time) => time - 1), 1000);
+const TimeToEgg = ({ showTimeToEgg, service }: TimeToEggProps) => {
+  const [{ context }] = useActor(service);
```

Comment:
There was actually no reason to have another interval running since we had one in the machine. Refactored to use the same one.